<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowCarsWindow</class>
 <widget class="QDialog" name="ShowCarsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="show_table">
     <property name="minimumSize">
      <size>
       <width>850</width>
       <height>300</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modelo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ano</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cilindrada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Potencia</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pre√ßo p/ Dia</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Matricula</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Disponibilidade</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="input_marca">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Digite a Marca do Veiculo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bot_marca">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Filtrar p/ Marca</string>
           </property>
           <property name="iconSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="input_modelo">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Digite o Modelo do Veiculo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bot_modelo">
           <property name="text">
            <string>Filtrar p/Modelo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="input_ano">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Digite o Ano do Veiculo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bot_ano">
           <property name="text">
            <string>Filtrar p/Ano</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="bot_avai">
           <property name="text">
            <string>Filtrar p/ Disponibilidade</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="bot_allcars">
         <property name="text">
          <string>Mostrar Todos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bot_menu">
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>*Para Alugar Clicar Duas Vezes em Qualquer Spec do Veiculo//Para Editar Apenas Mudar o Texto Desejado(Disponibilidade 0-&gt;Disponivel: 1-&gt;Alugado)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
